app/validators/claim/litigator_hardship_claim_validator.rb

```diff
@@ -0,0 +1,33 @@
+class Claim::LitigatorHardshipClaimValidator < Claim::BaseClaimValidator
+  include Claim::LitigatorCommonValidations
+
+  def self.fields_for_steps
+    {
+      case_details: %i[
+        case_type
+        court
+        case_number
+        case_transferred_from_another_court
+        transfer_court
+        transfer_case_number
+      ],
+      defendants: [],
+      offence_details: %i[offence],
+      # graduated_fees: %i[actual_trial_length total],
+      # miscellaneous_fees: %i[defendant_uplifts],
```

Comment:
does anything need doing with these?

Reply:
from the prototype it looks like miscellaneous fee is required. not sure if the `defendant_uplifts` is needed as it does not appear on the misc fee page for this or the lgfs final claim  type from which this was copied :(